ci/get-ocp-repo.sh

```diff
@@ -16,8 +17,15 @@ urls=(
 )
 
 dest=$1; shift
+variant=$1; shift
 
 rm -f "$dest"
 for url in "${urls[@]}"; do
     curl --fail -L "$url" >> "$dest"
 done
+# ART team does not set the Y stream in RHEL10 repo definition, so we have to add it back to be consistent
+# with our pipeline.
+# https://github.com/openshift/release/blob/master/core-services/release-controller/_repos/ocp-4.20-rhel10.repo
```

Comment:
I would maybe make a more explicit comment about how one OCP release is tied to one Y release of RHEL so specifying a Y stream in prow is redundant